test/README.md

```diff
@@ -35,3 +35,78 @@ Some examples of test harnesses using these tests, from the source repository:
 - [CST parse/stringify tests](https://github.com/messageformat/messageformat/blob/11c95dab2b25db8454e49ff4daadb817e1d5b770/packages/mf2-messageformat/src/cst/cst.test.ts)
 - [Data model stringify tests](https://github.com/messageformat/messageformat/blob/11c95dab2b25db8454e49ff4daadb817e1d5b770/packages/mf2-messageformat/src/data-model/stringify.test.ts)
 - [Formatting tests](https://github.com/messageformat/messageformat/blob/11c95dab2b25db8454e49ff4daadb817e1d5b770/packages/mf2-messageformat/src/messageformat.test.ts)
+
+## Test Functions
+
+As the behaviour of `:number`, `:datetime`, and other default message functions
+is dependent on locale-specific data and may vary between implementations,
+the following functions are defined for test use only:
+
+### `test:number`
+
+This function is valid both as a selector and as a formatter.
+The only option it recognizes is `fractionDigits`,
+all other option values passed to it are ignored.
```

Comment:
How would I use this?

A test like this still doesn't work:

```
You have {$num :number} elderberries.
```

I have to rewrite it to use `:test:number` and `:test:number` has different options from `:number`. Implementations cannot test full conformance with MF2 using the tests because default registry is included in full conformance.

Maybe a better approach would be to have an invariant locale or a test locale and to define what the outputs for that locale are. e.g. we could use the root locale (`und`) or possibly a patch on that (`und-x-test`) [note that pseudo-translation locales recommend themselves here as well--especially the potential for RTL pseudo-locales so both LTR and RTL messages can be tested]

Possible we should consider doing design first?